packages/core/src/networks/mainnet.ts

```diff
@@ -25,8 +25,8 @@ export const networks: Networks = {
     nativeBridgeUrl: 'https://gateway.optimism.io/welcome',
     waitConfirmations: 1
   },
-  xdai: {
-    name: chains.xdai.name,
+  gnosis: {
+    name: chains.gnosis.name,
     networkId: 100,
     publicRpcUrl: 'https://rpc.xdaichain.com',
```

Comment:
All `https://rpc.xdaichain.com` should be changed to `https://rpc.gnosischain.com`